.github/workflows/cron.yml

```diff
@@ -53,26 +59,49 @@ jobs:
           username: ${{ secrets.ECR_ACCESS_KEY_ID }}
           password: ${{ secrets.ECR_SECRET_ACCESS_KEY }}
 
-      - name: Upload assets to GHCR and ECR Public
+      - name: Install oras
+        run: |
+          curl -LO https://github.com/oras-project/oras/releases/download/v1.2.0/oras_1.2.0_linux_amd64.tar.gz
+          tar -xvf ./oras_1.2.0_linux_amd64.tar.gz          
+
+      - name: Upload assets to registries
         run: |
           lowercase_repo=$(echo "${{ github.repository }}" | tr '[:upper:]' '[:lower:]')
-          oras version
+          ./oras version
+          
+          echo "Starting artifact upload process..."
           
-          # Define an array of registry base URLs
-          registries=(
-            "ghcr.io"
-            "public.ecr.aws"
+          # Define an array of registry base URLs and their corresponding repository names
+          declare -A registries=(
+            ["ghcr.io"]="${lowercase_repo}"
+            ["public.ecr.aws"]="${lowercase_repo}"
+            ["docker.io"]="${lowercase_repo}"
```

Comment:
Can you please open another PR for this change?

Reply:
no problem